### DISCUSSION TITLE
Disable CSAT for Auto resolved conversations

### DISCUSSION THREAD

**Discussion by 1t5j0y** [chatwoot/chatwoot#4002] [Q&A] [answered]
I'm on version 1.22.0

I have a 30 day auto resolve conversation configuration at the account level and have enabled CSAT on my website inbox.

Is there a way to disable sending the customer satisfaction survey for conversations that have been auto-resolved since users are receiving the survey after 30 days while agents haven't been able to mark the conversations as resolved earlier due to various reasons such as lack of response etc.

**Comment by sojan-official** [comment, marked as answer]
@1t5j0y  its not possible at the moment. I have logged https://github.com/chatwoot/chatwoot/issues/4003  to keep track of this
